src/taco-cli/test/utils/httpServerFunction.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../../../typings/node.d.ts" />
+
+interface IHttpServerFunction {
+    (request: NodeJSHttp.ServerRequest, response: NodeJSHttp.ServerResponse): void;
+};
+
+export = IHttpServerFunction;
```

Comment:
left it as is, since this is test specific. sort of internal for the tests. if we move it to typedef, it suddenly becomes public to whole code..